module/VuFind/src/VuFind/ILS/Driver/KohaILSDI.php

```diff
@@ -1201,14 +1200,16 @@ public function getMyFinesILS($patron)
         $this->debug("ID: " . $rsp->{'borrowernumber'});
         $this->debug("Chrgs: " . $rsp->{'charges'});
 
+        $this->debug($rsp->{'fines'});
```

Comment:
Should this debug message have a label on it, or is it self-explanatory?

Reply:
Actually, I can drop it as I fixed the issue it was related to debugging.